lib/mvi/service.rb

```diff
@@ -78,6 +78,18 @@ def self.formatted_response(body)
         )
       }
     end
+
+    # MVI correlation id source id relationships:
+    # {source id}^{id type}^{assigning authority}^{assigning facility}^{id status}
+    # TODO(AJD): MVI team will be sending the mapping of system identifiers to
+    # va systems (e.g. 200VETS = vets.gov, 516 = ?) when we have that we can symbolize the keys
+    #
+    def self.map_correlation_ids(ids)
```

Comment:
Agree that it should be private. Need to think about this a bit more. Realize singletons are effectively the same as using only class methods, however not crazy about Klass.instance.foo stylistically 😃